migrations/1624695890253-CreateRfqmWorkerHeartbeatTable.ts

```diff
@@ -0,0 +1,37 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const TABLE_NAME = 'rfqm_worker_heartbeats';
+
+const rfqmWorkerHeartbeatTable = new Table({
+    name: TABLE_NAME,
+    columns: [
+        { name: 'address', type: 'varchar', isPrimary: true },
+        { name: 'timestamp', type: 'timestamptz', default: 'NOW()' },
+        { name: 'balance', type: 'numeric' },
```

Comment:
just curious, is this going to be `u256` or is it going to be a decimal number?

Reply:
BigNumber